src/_guides/libraries/c-interop.md

```diff
@@ -12,6 +12,13 @@ _FFI_ stands for [_foreign function interface._][FFI]
 Other terms for similar functionality include _native interface_
 and _language bindings._
 
+{{ site.alert.info }}
```

Comment:
Overall I think this may cause more questions than offer answers. Is there something we can link to regarding how to expose a C interface from these various languages?

Reply:
@mjohnsullivan any suggestions?

Reply:
I see some relevant C++ topics in Stackoverflow, such as https://stackoverflow.com/questions/2045774/developing-c-wrapper-api-for-object-oriented-c-code.

I haven't found anything similar for Swift.

https://en.wikipedia.org/wiki/Foreign_function_interface#By_language seems like it might be a good resource.

Reply:
@sjindel-google @dcharkes are you aware of any resources we can link to?

Reply:
For Go: https://golang.org/cmd/cgo/
For Swift: https://stackoverflow.com/questions/35106118/what-is-the-best-way-to-call-into-swift-from-c, not officially supported